cli/pcluster/config/pcluster_config.py

```diff
@@ -364,6 +385,8 @@ def __check_account_capacity(self):  # noqa: C901
             message = e.response.get("Error").get("Message")
             if code == "DryRunOperation":
                 pass
+            elif code == "UnsupportedOperation":
+                error(message.replace("CpuOptions", "disable_hyperthreading"))
```

Comment:
I think we should write our custom message instead of replacing `CpuOptions` from a message that can change in future.

Reply:
Ok, thanks for the detailed explanation! 👍  I agree we can keep the `message.replace()` for now.